README.md

```diff
@@ -197,11 +197,23 @@ With it you can request a certificate from ACM Private CA (PCA) where ACM genera
 ```bash
 ./cli.py request --domain "example.example.com" --base-url "https://abcde12345.execute-api.us-east-1.amazonaws.com/v1/request" --policy Default --arn "arn:aws:acm-pca:us-east-1:123456789000:certificate-authority/aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
 ```
+
+The output will be a certificate arn. e.g: `arn:aws:acm:us-east-1:123456789000:certificate/xxxxxxxx-yyyy-yyyy-yyyy-zzzzzzzzzzzz`.
+This certificate will also be listed in the AWS Console under Certificate Manager.
+
+
+
 Or you can request a certificate by providing your own CSR for the PCA to sign:
 ```bash
 ./cli.py issue --csr-path "/home/user/csr.pem" --base-url "https://abcde12345.execute-api.us-east-1.amazonaws.com/v1/request" --policy Default --arn "arn:aws:acm-pca:us-east-1:123456789000:certificate-authority/aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
 ```
 
+Because this command uses PCA to issue a certificate, it will not be listed within the AWS Console. To obtain the issued certificate run: 
```

Comment:
@cdmadrigal is that accurate that the certificate won't be visible in the AWS web console?  It has been awhile since I've worked with this solution but my recollection is that all certificates appeared in the web console regardless of how they were requested.  I agree that a second call is needed to get the actual certificate because the issue request only returns the ARN.

Reply:
@tr1ck3r The 'request' command does show you the certificates in the webUI, the 'issue' command does not. The only way you can see the certificate that you actually generated (without the CLI command to do so), is by running an audit report on your AWS PCA which developers shouldn't have access to. 

From AWS Documentation:
> You use the ACM Private CA API or AWS CLI to issue a private certificate. But if you do, you cannot use the ACM console, CLI, or API to view or export it. You already have the private key because you needed one to create the CSR (see the previous discussion). You can use the get-certificate command to retrieve the certificate details. You can also create an audit report to make sure that your certificate was issued.

https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaIssueCert.html